config/lib/seed.js

```diff
@@ -36,13 +41,13 @@ function checkUserNotExists (user) {
     var User = mongoose.model('User');
     User.find({username: user.username}, function (err, users) {
         if (err) {
-          reject(new Error('Database Seeding:\t\t\tFailed to find local account ' + user.username));
+          reject(new Error('Failed to find local account ' + user.username));
         }
 
         if (users.length === 0) {
           resolve();
```

Comment:
Shouldn't this resolve with something like `true` or `[]` or something more concrete than nothing?